batchflow/named_expr.py

```diff
@@ -401,6 +404,58 @@ def __repr__(self):
         return 'Unknown expression'
 
 
+class IF(NamedExpression):
+    """ Select either ``true`` or ``false``, based on ``condition``.
+    Useful for simple variables that change along the run of a pipeline.
+
+    Examples
```

Comment:
Not as in the numpy docstyle 
https://numpydoc.readthedocs.io/en/latest/format.html#examples

Reply:
And here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Pretty much the entire file is not in numpy docstyle, as well as Pipeline and some other files. [Local consistency is the most important](https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)